x-pack/plugins/alerting/server/authorization/alerting_authorization.ts

```diff
@@ -285,7 +286,7 @@ export class AlertingAuthorization {
     if (this.authorization && this.shouldCheckAuthorization()) {
       const { username, authorizedRuleTypes } = await this.augmentRuleTypesWithAuthorization(
         this.ruleTypeRegistry.list(),
-        [ReadOperations.Find],
+        [operation == null ? ReadOperations.Find : operation],
```

Comment:
If you are changing the name to `getAuthorizationFilter` (which makes sense to me), I would make the parameter `operation` non-optional and not default to `ReadOperations.Find`. Maybe maintain backwards compatibility by creating a `getFindAuthorizationFilter` fn that calls this fn and passes in `ReadOperations.Find`?

Reply:
Awesome will do! Thanks for the review @ymao1